docs/tasks/setup_all_or_nothing.md

```diff
@@ -0,0 +1,456 @@
+job1-6-m2qzw    0/1     ContainerCreating   0          1s
+job1-7-hgp9n    0/1     ContainerCreating   0          1s
+job1-8-ss248    0/1     ContainerCreating   0          1s
+job1-9-nwqmj    0/1     ContainerCreating   0          1s
+```
+
+Output when `job1` is running and `job2` is now unsuspended as `job` has all
+the required resources assigned:
+
+```
+NAME            READY   STATUS      RESTARTS   AGE
+job1-0-gc284    1/1     Running     0          9s
+job1-1-xz555    1/1     Running     0          9s
+job1-10-2ltws   1/1     Running     0          9s
+job1-11-r4778   1/1     Running     0          9s
+job1-12-xx8mn   1/1     Running     0          9s
+job1-13-glb8j   1/1     Running     0          9s
+job1-14-gnjpg   1/1     Running     0          9s
+job1-15-dzlqh   1/1     Running     0          9s
+job1-16-ljnj9   1/1     Running     0          9s
+job1-17-78tzv   1/1     Running     0          9s
+job1-18-4lhw2   1/1     Running     0          9s
+job1-19-hx6zv   1/1     Running     0          9s
+job1-2-hqlc6    1/1     Running     0          9s
+job1-3-zx55w    1/1     Running     0          9s
+job1-4-k2tb4    1/1     Running     0          9s
+job1-5-2zcw2    1/1     Running     0          9s
+job1-6-m2qzw    1/1     Running     0          9s
+job1-7-hgp9n    1/1     Running     0          9s
+job1-8-ss248    1/1     Running     0          9s
+job1-9-nwqmj    1/1     Running     0          9s
+job2-0-djnjd    1/1     Running     0          3s
+job2-1-trw7b    0/1     Pending     0          2s
+job2-10-228cc   0/1     Pending     0          2s
+job2-11-2ct8m   0/1     Pending     0          2s
+job2-12-sxkqm   0/1     Pending     0          2s
+job2-13-md92n   0/1     Pending     0          2s
+job2-14-4v2ww   0/1     Pending     0          2s
+job2-15-sph8h   0/1     Pending     0          2s
+job2-16-2nvk2   0/1     Pending     0          2s
+job2-17-f7g6z   0/1     Pending     0          2s
+job2-18-9t9xd   0/1     Pending     0          2s
+job2-19-tgf5c   0/1     Pending     0          2s
+job2-2-9hcsd    0/1     Pending     0          2s
+job2-3-557lt    0/1     Pending     0          2s
+job2-4-k2d6b    0/1     Pending     0          2s
+job2-5-nkkhx    0/1     Pending     0          2s
+job2-6-5r76n    0/1     Pending     0          2s
+job2-7-pmzb5    0/1     Pending     0          2s
+job2-8-xdqtp    0/1     Pending     0          2s
+job2-9-c4rcl    0/1     Pending     0          2s
+```
+
+Once `job1` completes it frees the resources required by `job2` to run its pods
+to make progress. Finally, all jobs complete.
+
+#### Cleanup
+
+Clean up the jobs by:
+```shell
+kubectl delete -f quick-job.yaml
+kubectl delete -f /tmp/job1.yaml
+kubectl delete -f /tmp/job2.yaml
+```
+
+## Drawbacks
+
+A drawback of using the `waitForPodsReady` is that admission of workloads may
+be unnecessarily slowed down by sequencing in case there is enough resources
+to support concurrent workload startup.
```

Comment:
```suggestion
When enabling `waitForPodsReady`, the admission of workloads may
be unnecessarily slowed down by sequencing in case the cluster has enough resources to support concurrent workload startup.
```